### ISSUE TITLE
Add "Ignored" label in DownloadCentral

### ISSUE THREAD

**Issue by markusmaresch** [gnosygnu/xowa#113]
This is a feature request - low priority, but nice to have. Admitted, this is bitching from a very high level :-)
In Download Central, the list of "Available" can get cumbersome to check:
- most users will only look for a small number of languages, like English and X
- some offerings in "Available" are outdated (don't want to download to make it go away ..)
- some downloads are really not of interest, yet are showing up until ... yes, what ? 
   -For instance "Wikidata-Wikitext" can be ignored for HTML DB's (see http://xowa.org/home/wiki/App/Databases/Overview) 
   - same for Wikinews - I don't care them (quickly outdated, not the goal of my Xowa use !)

Would it be possible to add a "Ignored" label below "Available" ? Dragging unwanted 'available's there might be a way to go.

Great project - great app !

**Comment by gnosygnu** [comment]
Hey, thanks for the feedback. 

Just to make sure, are you using version v3.10.4.1 or greater? v3.10.4.1 adds a few options to handle your points. See the screenshot below as well as the following:

> most users will only look for a small number of languages, like English and X

There is now a "Lang" and a "Type" drop-down. It defaults to All, but selecting "German" will only show the German wikis.

> some offerings in "Available" are outdated (don't want to download to make it go away ..)

v3.10.4.1 automatically removes outdated wikis (wikis which are no longer available in archive.org). If you have v3.10.4.1, you should only see the latest wiki (2016-11 English Wikipedia), not several (2016-11, 2016-10, 2016-09, etc.). If you're seeing multiple, can you let me know which wiki and which dates? (or post a screenshot)

> some downloads are really not of interest, yet are showing up until ... yes, what ?

There isn't a filter for "Wikidata-Wikitext" nor are there negative filters (show me everything except Wikinews). However, it does allow you to show only specific ones (like "only show me Wikipedia".

----

If you are running v3.10.4.1, then I guess there's still more that can be done. I do agree that the Download Central list can be cluttered even despite the above. I am planning to add new configuration settings which:

* Remember the last chosen Lang / Type (right now, it always defaults to all)
* Allow some basic way of selecting a group of specific langs or wiki types (something like "en;de" would only show English and German or "wikipedia;wiktionary;wikisource" to only show those these three wikis)
* Choose whether or not to show Category, Files, or Wikitext (not yet uploaded, but will be once the config setting is in place)

In theory, a new Ignored label can be added, but I think the above Config changes would accommodate the same need.

Let me know what you think. Thanks!

![xowa__download_central__filters](https://cloud.githubusercontent.com/assets/4994104/20994385/0890b148-bcbe-11e6-817a-18312cf42e8d.png)

**Comment by markusmaresch** [comment]
Thanks for the explanation. I am running 3.12.1.1 - and agree with what you are saying.
For me there is one left over: "English Wikipedia - Image Update (2016-09) [8.2 GB]"
This is from the time of transition to DownloadCentral, and I don't want to update (go backward) of course.
I am good to close this then.

**Comment by gnosygnu** [comment]
> For me there is one left over: "English Wikipedia - Image Update (2016-09) [8.2 GB]"

Ah, I didn't realize you were talking about a specific image update.

Let me think about this. I can add a "-" icon next to the "info" icon on the right. If I go this route, I'll do it whenever I add in the other config changes -- so I can group the Download Central changes in a single release.

Thanks.

**Comment by markusmaresch** [comment]
Please remove this one left over: "English Wikipedia - Image Update (2016-09) [8.2 GB]" in the DB.
Good to close. Thanks.

**Comment by gnosygnu** [comment]
> Please remove this one left over: "English Wikipedia - Image Update (2016-09) [8.2 GB]" in the DB.

Yeah, I'm changing Config options for the next release, but it'll take me a little while longer before you can customize the list on your device. I'll keep this ticket open for a few weeks until it's ready.

Thanks.

**Comment by gnosygnu** [comment]
v4.2.0 adds a "blue minus" icon to Download Central. This will move an item from the Available table to the Imported table

I'm going to mark this issue resolved. Let me know if there's anything else. Otherwise, I'll close it out in a few days.

**Comment by gnosygnu** [comment]
As mentioned above, this was resolved in v4.2. I'm marking the ticket closed. Let me know if there's anything else. Thanks.
